client/dive-common/components/ImportAnnotations.vue

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+import { useApi } from 'dive-common/apispec';
+import { usePrompt } from 'dive-common/vue-utilities/prompt-service';
+
+export default defineComponent({
+  name: 'ImportAnnotations',
+  props: {
+    datasetId: {
+      type: String,
+      default: null,
+    },
+    blockOnUnsaved: {
+      type: Boolean,
+      default: false,
+    },
+    buttonOptions: {
+      type: Object,
+      default: () => ({}),
+    },
+    menuOptions: {
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  setup(props, { emit }) {
+    const { openFromDisk, importAnnotationFile } = useApi();
+    const { prompt } = usePrompt();
+    const openUpload = async () => {
+      const ret = await openFromDisk('annotation');
+      if (!ret.canceled) {
+        const path = ret.filePaths[0];
+        let importFile = false;
+        if (ret.fileList?.length) {
+          importFile = await importAnnotationFile(props.datasetId, path, ret.fileList[0]);
+        } else {
+          importFile = await importAnnotationFile(props.datasetId, path);
+        }
+        if (importFile) {
+          emit('reimport-annotation-file');
+        } else {
+          const text = `Import of File ${path} failed`;
+          prompt({
+            title: 'Import Failed',
+            text,
+            positiveButton: 'OK',
+          });
+        }
+      }
+    };
+    return {
+      openUpload,
+    };
+  },
+});
+</script>
+
+<template>
+  <v-btn
+    class="ma-0"
+    v-bind="buttonOptions"
+    :disabled="!datasetId"
+    @click="openUpload"
+  >
+    <v-icon>
+      mdi-application-import
+    </v-icon>
+    <span
+      v-show="!$vuetify.breakpoint.mdAndDown || buttonOptions.block"
+      class="pl-1"
+    >
+      Import
+    </span>
+  </v-btn>
```

Comment:
wrap in a tooltip like the others?

Also, is there a way to make it clearer what this does?  I could imagine someone getting the idea that this would let them import new images or video.

Reply:
It may be time for that Data drop-down?  Any more text I add there is going to clutter it up.  The benefit is that import, download, clone are all one time actions that I can be hidden in a submenu.  I can work on something.